### PR TITLE
[7.0.x] GUVNOR-2782: Solve marshalling issues of ImpossibleMatchIssue

### PR DESCRIPTION
**Pull request by jomarko**
Please do not merge yet

### CODE REVIEW

File: drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/testutil/AnalyzerProvider.java [hunk 1]
```diff
@@ -42,6 +42,12 @@
 
     public AnalyzerProvider() {
 
+        factTypes.add( new FactTypes.FactType( "Applicant",
+                                               new HashSet<FactTypes.Field>() {{
```
Comment (triceo) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/testutil/AnalyzerProvider.java]: Unless you need to modify the `Set` later, `Collections.singleton(...)` will work just as well and will bring nicer code.

**Comment by jomarko** [comment]
Thanks @Rikkola. I just wanted wait for green (because of troubles during merge into master).

**Comment by manstis** [comment]
Erm.. @Rikkola this PR is for 7.0.x

**Comment by manstis** [comment]
@jomarko Please give the nod when you want this merged (was you taking @triceo recommendation; or keeping it consistent with the code in ```master```?)

**Comment by jomarko** [comment]
@manstis I incorporated @triceo's comment. However I applied it just on the commented line.

**Pull request merged**